RELEASING.md

```diff
@@ -51,7 +51,7 @@ scaladoc generation build task, which is part of the release.
 ## Snapshot releases
 
 Snapshot releases are created from `main` and published to
-https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/akka/
+https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/pekko/
```

Comment:
can you remove this section about snapshots - we can replace it when we have nightly builds

Reply:
Done